lib/tmuxinator/cli.rb

```diff
@@ -169,7 +169,8 @@ def config_path(name, local = false)
       def generate_project_file(name, path)
         template = Tmuxinator::Config.default? ? :default : :sample
         content = File.read(Tmuxinator::Config.send(template.to_sym))
-        erb = Erubis::Eruby.new(content).result(binding)
+        erubi_content = Erubi::Engine.new(content).src
+        erb = binding.instance_eval(erubi_content)
```

Comment:
What does using `instance_eval` instead of `eval` do? `binding` is a new instance every time it's called.

Reply:
> What does using `instance_eval` instead of `eval` do? `binding` is a new instance every time it's called.

It's been so long since I last looked at this that I'm afraid the best I can remember was something about limiting scope and being able to dump the object for debugging.

Reply:
It makes sense to me that `instance_eval` is safer/more limited scope.

I notice we're not consistent with `eval` vs. `instance_eval` in this PR, [like in project.rb](https://github.com/tmuxinator/tmuxinator/pull/793/files#diff-e013992ae474239aef1e409cb960c112e5c981d8c6fedacb5451245b6fbfa289R107). Maybe we should be using `instance_eval` there as well?

Maybe unrelated, but [this SO answer](https://stackoverflow.com/a/46567196) shows how `instance_eval` can be used with a context object and instance variables (i.e. `<%= @foo %>`) in the template, which is common in Rails, for example. I don't think we're doing any of that in this repo. 🤔